easybuild/framework/easyconfig/default.py

```diff
@@ -158,6 +158,11 @@
 
     # MODULES easyconfig parameters
     'whatis': [None, "List of brief (one line) package description entries", MODULES],
+    'usage': [None, "Usage instructions for the software", MODULES],
+    'docpaths': [None, "List of paths of installed documentation relative to package root dir", MODULES],
+    'docurls': [None, "List of URLs to package documentation", MODULES],
+    'support': [None, "Package-specific support/bug report address", MODULES],
+    'contact': [None, "Site contact for the package", MODULES],
```

Comment:
hmm, this is getting a bit messy, is it not?

we should try and sort these alphabetically, although maybe also keep related ones together in groups?